sass/_header.scss

```diff
@@ -0,0 +1,58 @@
+
+.bg__color--blue {
+	.body__container--header {
+		display: flex;
+        flex-direction: column;
+		// justify-content: space-between;
```

Comment:
@Jelena-ag wszystkie komentarze raczej usuwamy, jeśli ten zakomentowany kod jest Ci potrzebny do dalszej pracy, to można go sobie np. zestashować i potem wyciągnąć na nowym branchu.

Reply:
u mnie jest odkomentowane (nie mam komentarzy teraz), jeżeli wejć po przez Githuba na ten branch/folder sass/plik _navigation.scss, to tam też nie ma komentarza.

Reply:
Komentarze zostały w _header.scss, a nie w _navigation :)

Reply:
ok, rozumiem. Header będę dorabiać w już w odpowiedniej dla niego sekcji.